### PR TITLE
Update ingest-data-no-code.md

### PR DESCRIPTION
**Pull request by belkiso**
Update on Event Hub namespace/Evet Hub creation links

### CODE REVIEW

File: data-explorer/ingest-data-no-code.md [hunk 1]
```diff
@@ -402,9 +402,9 @@ Azure diagnostic settings enable exporting metrics and logs to a storage account
 
     [![Deploy to Azure button.](media/ingest-data-event-hub/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fquickstarts%2Fmicrosoft.eventhub%2Feventhubs-create-namespace-and-eventhub%2Fazuredeploy.json)
 
-1. Create an Event Hubs namespace and an event hub for the diagnostic logs.
-
-    ![Event hub creation.](media/ingest-data-no-code/event-hub.png)
+1. Follow link to create Event Hub namespace and an Event Hub for the diagnostic logs.
+   
+    [Event hub namespace and Event Hub creation](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create)
```
Comment (YaelSchuster) [inline data-explorer/ingest-data-no-code.md:407]:
```suggestion
    [Event hub namespace and Event Hub creation](/azure/event-hubs/event-hubs-create)
```

File: data-explorer/ingest-data-no-code.md [hunk 2]
```diff
@@ -402,9 +402,9 @@ Azure diagnostic settings enable exporting metrics and logs to a storage account
 
     [![Deploy to Azure button.](media/ingest-data-event-hub/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fquickstarts%2Fmicrosoft.eventhub%2Feventhubs-create-namespace-and-eventhub%2Fazuredeploy.json)
 
-1. Create an Event Hubs namespace and an event hub for the diagnostic logs.
-
-    ![Event hub creation.](media/ingest-data-no-code/event-hub.png)
+1. Follow link to create Event Hub namespace and an Event Hub for the diagnostic logs.
```
Comment (YaelSchuster) [inline data-explorer/ingest-data-no-code.md:405]:
```suggestion
1. Follow the following link to create an Event Hub namespace and an Event Hub for the diagnostic logs.
```

**Comment by YaelSchuster** [comment]
Approved by ADX content team
#sign-off

**Comment by PRMerger18** [comment]
@YaelSchuster: I'm sorry - only the author of this article, @orspod, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Pull request merged**